src/main/java/teammates/ui/controller/InstructorCourseEditSaveAction.java

```diff
@@ -22,9 +22,7 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         InstructorAttributes instructor = logic.getInstructorForGoogleId(courseId, account.googleId);
         new GateKeeper().verifyAccessible(instructor, logic.getCourse(courseId), Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COURSE);
         
-        CourseAttributes courseToEdit = new CourseAttributes();
-        courseToEdit.id = courseId;
-        courseToEdit.name = courseName;
+        CourseAttributes courseToEdit = new CourseAttributes(courseId, courseName);
```

Comment:
@damithc I think we can either use
- immutable approach: constructors + getters, or 
- mutable approach: setters + getters

I think the immutable approach is better because it is more consistent with current implementation:
- I looked at a few of the other subclasses of `EntityAttributes`,
- the current implementation of `CourseAttributes`, `StudentAttributes`, `CommentAttributes` and `FeedbackSessionAttributes` are along the lines of immutable approach as the constructors are appropriately overloaded and there are getters but no setters.


(After refactoring `CourseAttributes`, I'll create a new issues for each of the other subclasses of `EnitityAttributes` that similarly have public member variables that actually requires sanitization.)